src/Apps/Collect/CollectApp.tsx

```diff
@@ -108,7 +105,7 @@ export const CollectAppFragmentContainer = createFragmentContainer(CollectApp, {
         page: { type: "Int" }
         dimension_range: { type: "String" }
       ) {
-      filter_artworks(aggregations: $aggregations, size: 0) {
+      filter_artworks(aggregations: $aggregations, sort: $sort) {
```

Comment:
This component fetches 30 artworks to present enriched acquireable product data with `jsonLD` (and filters out ones that aren't acquireable, so it may present less than 30 to google). Should these be the same 30 artworks that the page is rendering? (This is all happening in https://github.com/artsy/reaction/blob/befdd355f160b717ab6760005600bedd7941adf7/src/Apps/Collect/Components/Seo/SeoProductsForArtworks.tsx#L29-L32 ).

Based on that comment, I would assume yes, which means all of the filter arguments should be passed in as well (not just `sort`, but `sort` at least matches the initial /collect route). Without the `sort`, for instance, you get a bunch of random artworks from ES, few of which are acquireable (and so, inspecting the page, you only have a couple of the 'product' JSON groups).

Passing in `sort` is an improvement, and I _think_ all of the arguments [below](https://github.com/artsy/reaction/pull/2639/files#diff-5cec8556ebdc7d774aaeeb67e4cf2638R113) should be passed in.

Reply:
Right, ideally there should be the same artworks that are displayed on the page. The tricky part here though, is that not all artworks have all the meta fields required by Google, and if an artwork does not meet their requirements, it would cause an error (usually testable on [their testing tool](https://search.google.com/structured-data/testing-tool/u/0/) I wonder how much this would affect SEO, but it's worth re-visiting for sure.